src/sagas/vmChanges.js

```diff
@@ -200,6 +200,7 @@ function* composeProvisionSourceTemplate ({ vm, basic, disks }) {
 
       // did the storage domain change?
       if (disk.storageDomainId !== templateDisk.get('storageDomainId')) {
+        vmRequiresClone = true
```

Comment:
Can we unit test this saga? Just the case we are modifying.

Reply:
Probably but the infrastructure isn't in place to do it.  Maybe a followup PR?  I would prefer to not spend that time at the moment.